tests/test_jointcal_cfht_minimal.py

```diff
@@ -126,70 +129,14 @@ def test_jointcalTask_2_visits_photometry_magnitude(self):
                    'photometry_final_ndof': 1
                    }
 
-        self._runJointcalTask(2, metrics=metrics)
+        self._runGen3Jointcal("lsst.obs.cfht.MegaPrime", "MegaPrime",
+                              configOptions=configOptions, metrics=metrics)
 
     def test_jointcalTask_fails_raise(self):
         """Raise an exception if there is no data to process."""
-        self.config = lsst.jointcal.jointcal.JointcalConfig()
-        self.config.setDefaults()
-        self.config.photometryModel = "simpleFlux"
-        self.config.sourceSelector['astrometry'].minSnr = 10000
-        self.config.doAstrometry = False
-
-        # The output repo is named after this method.
-        caller = self.id()
-        nCatalogs = 2
-        visits = '^'.join(str(v) for v in self.all_visits[:nCatalogs])
-        output_dir = os.path.join('.test', self.__class__.__name__, caller)
-        test_config = os.path.join(lsst.utils.getPackageDir('jointcal'), 'tests/config/config.py')
-        self.configfiles = [test_config] + self.configfiles
-        args = [self.input_dir, '--output', output_dir,
-                '--clobber-versions', '--clobber-config', '--configfile', *self.configfiles,
-                '--doraise',
-                '--id', 'visit=%s'%visits]
-        args.extend(self.other_args)
+        self.configfiles.append(os.path.join(self.path, 'config/minSnr.py'))
         with self.assertRaisesRegex(RuntimeError, "No data to process"):
-            jointcal.JointcalTask.parseAndRun(args=args, doReturnResults=True, config=self.config)
-
-    def test_jointcalTask_fails_no_raise(self):
```

Comment:
Is it safe to remove this test completely?

Reply:
Yes: this is testing gen2-only functionality here. The `test_jointcalTask_fails_raise` test demonstrates that `JointcalTask` will raise if there is no data, but whether it raises in a `pipetask` context, or just goes on to the next quantum and logs a failure is entirely up to the pipetask configuration and not something I can test within jointcal itself.